src/olympia/files/utils.py

```diff
@@ -329,25 +314,22 @@ def apps(self):
             < VersionString(amo.DEFAULT_WEBEXT_MIN_VERSION)
         )
         if unsupported_no_matter_what:
-            msg = gettext('Lowest supported "strict_min_version" is 42.0.')
+            msg = gettext(
+                'Lowest supported "strict_min_version" is '
+                f'{amo.DEFAULT_WEBEXT_MIN_VERSION}.'
```

Comment:
I'd be surprised if this l10n string was correctly extracted. It should probably have a parameter.

Reply:
ah right, forgot about that.